styles/style.css

```diff
@@ -0,0 +1,79 @@
+@import url("fonts.css");
+* {
+  margin: 0;
+  font-family: "Roboto", "sans-serif";
```

Comment:
You should use sans-serif without quotes
Look at examples [link](https://developer.mozilla.org/ru/docs/Web/CSS/font-family)